File: Vicopo/Vicopo.py
import urllib, json

class Vicopo:
    @staticmethod
    def get(search, protocol = 'https'):
        response = urllib.urlopen(protocol + '://vicopo.selfbuild.fr/search/' + str(search))
        return json.loads(response.read())
    @staticmethod
    def http(search):
        return Vicopo.get(search, 'http')
    @staticmethod
    def https(search):
        return Vicopo.get(search, 'https')

File: src/Vicopo/Vicopo.php
<?php

namespace Vicopo;

class Vicopo {

	static protected function get($search, $protocol = 'https') {
		$vicopoUrl = $protocol . '://vicopo.selfbuild.fr/search/' . urlencode($ville);
		$json = @json_decode(file_get_contents($vicopoUrl));
		if (!is_object($json) || !isset($json->cities)) {
			throw new Exception("No valid answear found", 1);
		}
		return $json->cities;
	}

	static public function http($search) {
		return static::get($search, 'http');
	}

	static public function https($search) {
		return static::get($search, 'https');
	}
}
